Lib/json/tool.py

```diff
@@ -68,12 +72,42 @@ def main():
             outfile = open(options.outfile, 'w', encoding='utf-8')
         with outfile:
             for obj in objs:
-                json.dump(obj, outfile, **dump_args)
+                if with_colors:
+                    json_str = json.dumps(obj, **dump_args)
+                    outfile.write(colorize_json(json_str))
+                else:
+                    json.dump(obj, outfile, **dump_args)
                 outfile.write('\n')
     except ValueError as e:
         raise SystemExit(e)
 
 
+color_pattern = re.compile(r'''
+    (?P<string>"(.*?)")     |   # String
```

Comment:
This has some hiccups with escaped quotes:

![image](https://github.com/user-attachments/assets/60c4995c-5aef-4be4-a10c-281e53488b5b)
We could use a negative look-behind to make sure the terminating quote isn't escaped: 
```suggestion
    (?P<string>"(.*?)(?<!\\)")     |   # String
```

yields:

![image](https://github.com/user-attachments/assets/55bc031a-69b3-4992-9b44-5d569b2616b5)

Reply:
I went with `"(\\"|[^"])*?"` which should work as well.